lib/api/controllers/collectionController.js

```diff
@@ -126,7 +139,7 @@ class CollectionController {
             this.kuzzle.pluginsManager.trigger('log:info', `Validation specification for ${specLogName} is about to be stored.`);
           });
 
-          return Promise.all(promises);
+          return Bluebird.all(promises);
         }
 
         this.kuzzle.pluginsManager.trigger('validation:error', response.description);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 85 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)